### ISSUE TITLE
Gradle intergration runs `build` task that usually triggers test

### ISSUE THREAD

**Issue by jreznot** [con-ceptive/quarkus-intellij-plugin#3]
If compile before launch flag for Gradle run configuration is enabled the plugin runs `clean` and then `build`. In Gradle build triggers `test` so this could be a very long operation. It would be better to use `assemble` instead.

Typical `build` flow:
```
:compileJava 
:processResources 
:classes 
:jar 
:assemble 
:compileTestJava 
:processTestResources 
:testClasses 
:test 
:check 
:build 
```

Typical `assemble` flow:
```
:compileJava 
:processResources 
:classes 
:jar 
:assemble 
```

I believe `test` is not essential for Running the application.

**Comment by wglanzer** [comment]
That's a good point, I'll fix this asap - thank you for pointing this out 👍

**Comment by wglanzer** [comment]
Fixed in version 1.1.1
